specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -377,7 +377,14 @@ Unit and integration tests will be introduced by following the official Airflow
 
 The communication between VDK Provider and Control Service will be secured through the
 use of HTTPS and OAuth 2.0. The most commonly used OAuth 2.0 flows will be supported (
-e.g. Client Credentials Flow, Refresh Token Flow, etc.).
+e.g. Client Credentials Flow, Refresh Token Flow, etc.) by using `vdk-control-cli`'s authentication
```

Comment:
don't say 'e.g' (for example) . Be specific. 
We do not support Client Credentials Flow as defined in https://datatracker.ietf.org/doc/html/rfc6749#section-4.4  ? (or described more simply here - https://oauth.net/2/grant-types/client-credentials/)  . Unless I am having a brain check out now (which happens to me). 

--
We do support Refresh token Grant (or flow) - it's implemented [here](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-control-cli/src/vdk/internal/control/auth/auth.py#L180)

We do support  Authorization Code Flow with Proof Key for Code Exchange (PKCE) - it's implemented [here](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-control-cli/src/vdk/internal/control/auth/redirect_auth.py) 

And we do support this (nonofficial - in other words, it does not conform to any RFC spec) API Key  flow - implemented [here](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-control-cli/src/vdk/internal/control/auth/apikey_auth.py) - which is somewhat similar to refresh token grant (and might be used for that actually). 

We also support kerberos. But as we will not use it in Airflow , let's not think about it.

We definitely need to document those. I don't believe we have good documentation. I can certainly help with that. 

--
Now that it's clear what we support currently in VDK. I believe we want to support this API key login type , right? 
This is probably what you mean by refresh token flow ? Is that right?